backend-project/small_eod/cases/tests/test_admin.py

```diff
@@ -0,0 +1,29 @@
+from django.test import TestCase
+from django.urls import reverse
+from ...users.mixins import AuthenticatedMixin
+
+
+class TestAddAdminMixin:
+    @property
+    def app_label(self):
+        raise NotImplementedError()
+
+    @property
+    def model_name(self):
+        raise NotImplementedError()
+
+    def get_url(self):
+        return reverse(f"admin:{self.app_label}_{self.model_name}_add")
+
+
+class TestCaseAddAdmin(TestAddAdminMixin, AuthenticatedMixin, TestCase):
+    app_label = "cases"
+    model_name = "case"
```

Comment:
Obecnie często przekazujemy klasy mixinach dla testów. Proponuje zachować ten interfejs, zatem przyjąć to `Case`.